providers/implementations/ciphers/cipher_chacha20.c

```diff
@@ -64,6 +65,19 @@ static void chacha20_freectx(void *vctx)
     }
 }
 
+static void *chacha20_dupctx(void *vctx)
+{
+    PROV_CHACHA20_CTX *ctx = (PROV_CHACHA20_CTX *)vctx;
+    PROV_CHACHA20_CTX *dupctx = NULL;
+    if (ctx != NULL) {
```

Comment:
Code style nit: one empty line between variable declarations and other statements

Reply:
fixed